label_studio/core/settings/label_studio.py

```diff
@@ -1,65 +1,65 @@
 """This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
 """
-import os
-import pathlib
+import json
 
-from core.settings.base import *
+from core.settings.base import *  # noqa
 
-DJANGO_DB = get_env('DJANGO_DB', DJANGO_DB_SQLITE)
-DATABASES = {'default': DATABASES_ALL[DJANGO_DB]}
+DJANGO_DB = get_env('DJANGO_DB', DJANGO_DB_SQLITE)  # noqa: F405
```

Comment:
Could we turn off F405 for this whole file (and all the other django settings files?)